kolibri/content/api.py

```diff
@@ -311,3 +312,14 @@ class FileViewset(viewsets.ModelViewSet):
 
     def get_queryset(self):
         return models.File.objects.all()
+
+
+class ChannelFileSummaryViewSet(viewsets.ViewSet):
```

Comment:
This could also be a detail view off the main ChannelMetaDataViewSet - but this is also fine.